lib/domain-page/ensService.ts

```diff
@@ -101,13 +106,39 @@ export const getENSDomainDataThroughSubgraph = async (
   return data;
 };
 
+function getParent(domain: string): string {
+  const parts = domain.split(".");
+
+  // remove the first part and return the remaining domain
+  return parts.slice(1).join(".");
+}
+
+const getBasicENSDomainData = async (name: string): Promise<DomainData> => {
+  const domainAdd = await getResolver(publicClient, { name });
+  const domainOwner = await getOwner(publicClient, { name });
+  const wrapperData = await getWrapperData(publicClient, { name });
+
+  return {
+    owner: domainOwner?.owner ?? "0x",
+    parent: getParent(name),
+    subdomains: [],
+    subdomainCount: 0,
+    resolver: {
+      id: "",
+      address: domainAdd as Address,
```

Comment:
Can you add a if (!isAddress(domainAdd)) block above, throwing error if needed?

Reply:
`domainAdd as Address` will already throw an error if `domainAdd` is not an `Address`

Reply:
Oh shoot, really?

I didn't know it worked this way, can you screenshot it for me, please?

Reply:
<img width="1100" alt="image" src="https://github.com/user-attachments/assets/deec8c8a-8a2d-44d4-adf2-8e7801e93b00">
<img width="502" alt="image" src="https://github.com/user-attachments/assets/00f0e240-f802-4f5a-be1e-cdbc108a61a7">

Reply:
@FrancoAguzzi Actually, you're right. That error was thrown by the Front End lol

Reply:
@FrancoAguzzi I've added a `(!isAddress(domainAdd))` block above 🤝🏻